ephemeris/shed_tools.py

```diff
@@ -363,6 +369,27 @@ def _parser():
         help="This updates all tools in Galaxy to the latest revision."
              "Use shed-tools update --help for more information",
         parents=[common_arguments])
+
+    for command_parser in [update_command_parser, install_command_parser]:
+        command_parser.add_argument(
+            "--test",
+            action="store_true",
+            dest="test",
+            help="Run tool tests on install tools, requires Galaxy 18.05 or newer."
```

Comment:
This requirement should be hardcoded somewhere
`gi.config.get_config()["version_major"]` returns the version.

Reply:
I don't know about this - Nate had mentioned wanting to get this on usegalaxy.org and we were intending to maybe place the Galaxy-side APIs in the usegalaxy branch to allow this testing against our variant of 18.01. I guess we could just hack up ephemeris though if we do get around to setting this up. I'll try to find a spot to make this change.